client/Log.cpp

```diff
@@ -72,6 +72,46 @@ std::string FormatGuid(const GUID& guid)
 	return ss.str();
 }
 
+std::string FormatTransportLayerCommand(TransportLayerCommand command)
+{
+	switch (command)
+	{
+	case TransportLayerCommand::JoinRequest:
+		return "Join Request";
+	case TransportLayerCommand::JoinGranted:
+		return "Join Granted";
+	case TransportLayerCommand::JoinRefused:
+		return "Join Refused";
+	case TransportLayerCommand::StartGame:
+		return "Start Game";
+	case TransportLayerCommand::SetPlayersList:
+		return "Set Players List";
+	case TransportLayerCommand::SetPlayersListFailed:
+		return "Set Players List Failed";
+	case TransportLayerCommand::UpdateStatus:
+		return "Update Status";
+	case TransportLayerCommand::HostedGameSearchQuery:
+		return "Hosted Game Search Query";
+	case TransportLayerCommand::HostedGameSearchReply:
+		return "Hosted Game Search Reply";
+	case TransportLayerCommand::GameServerPoke:
+		return "Game Server Poke";
+	case TransportLayerCommand::JoinHelpRequest:
+		return "Join Help Request";
+	case TransportLayerCommand::RequestExternalAddress:
+		return "Request External Address";
+	case TransportLayerCommand::EchoExternalAddress:
+		return "Echo External Address";
+	default:
+		return "Unknown Transport Layer Command";
+	}
+}
+
+std::string FormatTransportLayerCommandIncludeIndex(TransportLayerCommand command)
+{
+	return FormatTransportLayerCommand(command) +  " (" + std::to_string(static_cast<int>(command)) + ")";
```

Comment:
Optional consideration: I might consider swapping the order of these two, particularly considering the context it is printed:
"commandType: ..."
"packet.commandType = ..."

```suggestion
	return std::to_string(static_cast<int>(command)) +  " (" + FormatTransportLayerCommand(command) + ")";
```

That might also allow printed text to line up nicer, since the numeric portion is more consistent in width than the text representation.